CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 # Changelog
 
-## [0.0.8]
+## [0.0.9]
+
+### Fixed
+* Fixed issue with Clustergram rows and columns reordering incorrectly
+  on the heatmap when precomputed traces are used to generate the
+  figure.
+
+### Changed
+* Changed Clustergram to only return a figure by default, so that
+  values no longer need to be unpacked.
+
+### Added
+* Added to Clustergram the ability to generate a "curves dictionary"
```

Comment:
I think it should be referred to as a "curve dictionary" (just like a real adjective, the "noun as adjective" is invariable; reference I could find very quickly: https://www.englishclub.com/grammar/nouns-adjective.htm).